packages/lisk-p2p/test/unit/error.ts

```diff
@@ -0,0 +1,60 @@
+/*
+ * Copyright © 2018 Lisk Foundation
+ *
+ * See the LICENSE file at the top-level directory of this distribution
+ * for licensing information.
+ *
+ * Unless otherwise agreed in a custom licensing agreement with the Lisk Foundation,
+ * no part of this software, including this file, may be copied, modified,
+ * propagated, or distributed except according to the terms contained in the
+ * LICENSE file.
+ *
+ * Removal or modification of this copyright notice is prohibited.
+ *
+ */
+import { expect } from 'chai';
+import { NotEnoughPeersError } from '../../src';
+
+describe('#not enough peers errors module', () => {
```

Comment:
Top title should be the module name.
Something like `Error Module` please check other tests?

Reply:
Probably you need one more outer `describe`